README.md

```diff
@@ -3,6 +3,8 @@
 These samples demonstrate integration between the Q# language infrastructure/drivers and other languages. 
 Currently present are the following integrations:
 
+![Build](https://github.com/qsharp-community/qsharp-integrations/workflows/Build/badge.svg)
```

Comment:
I've always wondered how this works? Does this image auto-update based on the branch/revision?

Reply:
![image](https://user-images.githubusercontent.com/6486256/87884928-92b0f680-c9c6-11ea-81f0-e61b1a2d50be.png)
GitHub will auto update the image with the status of the particular workflow! You can generate these badges with the button you see here on the right when you look at a particular workflow under actions for a repo.

Reply:
Thanks, @crazy4pi314